streams/src/main/java/org/apache/kafka/streams/processor/internals/DefaultStateUpdater.java

```diff
@@ -60,6 +60,7 @@ private class StateUpdaterThread extends Thread {
         private final ChangelogReader changelogReader;
         private final AtomicBoolean isRunning = new AtomicBoolean(true);
         private final Map<TaskId, Task> updatingTasks = new ConcurrentHashMap<>();
+        private final Map<TaskId, Task> pausedTasks = new ConcurrentHashMap<>();
```

Comment:
I moved the map of paused tasks from the state update to the state updater thread since it actually is -- as the map of updating tasks -- an internal data structure of the thread.